test/Bin/GameTest/GameTests_1500.cpp

```diff
@@ -160,3 +160,12 @@ GAME_TEST(Issues, Issue1597) {
     EXPECT_NE(attrEnchantmentsNum, 0);
     EXPECT_NE(specialEnchantmentsNum, 0);
 }
+
+// 1600
+
+GAME_TEST(Issues, Issue1671) {
+    // Falling from height outdoors onto models doesnt cause damage
+    auto health = tapes.totalHp();
+    test.playTraceFromTestData("issue_1671.mm7", "issue_1671.json");
+    EXPECT_LT(health.back(), health.front()); // party has taken damage from fall
```

Comment:
Let's also add a check for fall? There was a character expression for this I think.

And a check that we end up on a model and not on the ground, if it's doable.